README.md

```diff
@@ -1,11 +1,64 @@
 # Description
 Coforma's response to the SWIFT technical challenge
 
-## Local Setup
+## TODO: Table of Contents
+
+- TODO: Links here
+
+## Getting Started
+
+### One time only
+
+Before developing locally, you'll need to install some tools. You may have some of these already, or you may not.
+
+- Install [nvm](https://github.com/nvm-sh/nvm):
+    ```bash
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+    ```
+
+- Use nvm to install specified version of [node](https://nodejs.org/en):
+    ```bash
+    cd <project-directory>
+    nvm use
+    ```
+- Install [yarn](https://classic.yarnpkg.com/en/docs/install):
+
+    ```bash
+    brew install yarn
+    ```
+
+- Install [pre-commit](https://pre-commit.com/#install):
+
+    ```bash
+    brew install precommit
+    ```
+     OR
+    ```bash
+    pip install precommit
+
+    ```
+
+- Install pre-commit [git hook scripts](https://pre-commit.com/#3-install-the-git-hook-scripts):
+
+    ```bash
+    cd <project-directory>
+    pre-commit install
+    ```
+
+### Before making commits
 
-###
 TODO: Link to signed commits, with process
 
+### Running the local development server
+
+To run the local development server:
+
+```bash
+yarn dev
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the locally running application.
+
 ### Using adr-tools
 
 This repository makes use of [adr-tools](https://github.com/npryce/adr-tools/tree/master) to record architectural decisions as part of the code base.
```

Comment:
Can you combine this with the one time install before you get started stuff

Reply:
Added installation stuff to the install area, but kept the rest of this section separate.